benchmarks/Grpc.AspNetCore.Performance/Grpc.AspNetCore.Performance.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
naming nit:  if the plan for this project is to be mostly about BenchmarkDotNet microbenchmarks, should it be called Grpc.AspNetCore.Microbenchmarks? (it's a more descriptive name and more aligned with existing terminology in grpc codebase).

Reply:
It will get renamed to that with a later PR. It's not quite ready https://github.com/aspnet/grpc-dotnet/pull/2/files

Reply:
Fine to leave for later.